README.md

```diff
@@ -228,6 +228,7 @@ For all triggers:
   You should format dates as YYYY-MM-DD, although other date formats may work (under the hood, this uses Date.parse).
 * `minutes_interval` — The repeat interval in minutes.
 * `minutes_duration` — The duration in minutes, needs to be greater than the minutes_interval.
+* `random_minutes_interval` — The delay time that is randomly added to the start time of the trigger.
```

Comment:
Is the default 0?

Reply:
Can we use 0?

Reply:
Yes, the default is zero. I haven't tried to set it to zero (the default value is actually an empty string which it treats as a zero). Will test that now!

Reply:
Ensured the unit tests verify setting zero - it turns out that if you try setting it to zero it returns an empty string.